wgpu-core/src/lib.rs

```diff
@@ -166,14 +187,19 @@ impl Drop for MultiRefCount {
 struct LifeGuard {
     ref_count: Option<RefCount>,
     submission_index: AtomicUsize,
+    #[cfg(debug_assertions)]
+    pub(crate) label: Label<'static>,
```

Comment:
any reason to not make it a `String`?

Reply:
So yeah, since I noticed this `Label` typed used elsewhere, I figured might as well use it here too. The main benefit is that some of the internal labels don't need to be constructed as `String`, but can utilize the `Cow<'static, str>` here, for example in `swap_chain_get_current_texture_view` the label for the `TextureView`. 

If it made a heap allocation here, it would be tempting to leave it be a empty string (which I think doesn't heap allocate). But it would actually be nice to have the internally created resource to be labeled too.

But this is a tiny thing, I'm fine plain 'ol `String` as well.

Reply:
I think we should be able to deal with `swap_chain_get_current_texture_view` later. For now, just make it something like
```rust
if cfg!(debug_assertions) { "swapchain".to_string() } else { String::new() }
```